x-pack/plugins/fleet/server/services/actions/types.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface BulkCreateResponse {
+  status: 'success' | 'failed' | 'mixed';
+  items: Array<{
+    status: 'success' | 'error';
+    // action_id
```

Comment:
nit: should be removed

Reply:
updated the comment 5f9bec8de9a158b575f9ce4b1ca73545a1901047